notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb

```diff
@@ -76,12 +76,11 @@
         "\n",
```

Comment:
<p>Remove all <code>Learn more about</code>s. Instead add a line at the end of this cell as follows:</p><p>Learn more about [Google Cloud Pipeline Evaluation Components](https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.17/google_cloud_pipeline_components.experimental.evaluation.html#google_cloud_pipeline_components.experimental.evaluation.EvaluationDataSamplerOp)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>done</p>